.github/workflows/test_size.yml

```diff
@@ -20,8 +20,10 @@ jobs:
         ]
         config: [
             Durandal,
-            MatekF405,
-            Pixhawk1-1M
+            VRUBrain-v51,
```

Comment:
i much prefer the 3 existing ones, as they represent the 3 main classes of boards that users use

Reply:
I'm only replacing the MatekF405 with the VRUBrain-v51 - how is that not in the same class?

Reply:
VRUBrain-v51 isn't on the market anymore when MatekF405 is. IMHO, we should maybe discontinue VRUBrain